### DISCUSSION TITLE
suntime missing

### DISCUSSION THREAD

**Discussion by kobius77** [mcguirepr89/BirdNET-Pi#886] [Q&A] [answered]
[got birdnetpi running in a debian11 vm](https://github.com/mcguirepr89/BirdNET-Pi/discussions/251#discussioncomment-5784434) and everything seems to work fine but the 'species stats' which throws me this:

![image](https://user-images.githubusercontent.com/3143680/235861775-b92a16c4-d8e3-48a8-b2bd-9b7eaa97f49c.png)

but the module suntime is listed by `pip3 list`:

```
Package          Version
---------------- --------------
certifi          2020.6.20
chardet          4.0.0
httplib2         0.18.1
idna             2.10
pip              20.3.4
pycurl           7.43.0.6
PySimpleSOAP     1.16.2
python-apt       2.2.1
python-dateutil  2.8.2
python-debian    0.1.39
python-debianbts 3.1.0
reportbug        7.10.3+deb11u1
requests         2.25.1
setuptools       52.0.0
six              1.16.0
suntime          1.2.5
urllib3          1.26.5
wheel            0.34.2
```

full error:
`File "/home/pi/BirdNET-Pi/birdnet/lib/python3.9/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 565, in _run_script
    exec(code, module.__dict__)
File "/home/pi/BirdNET-Pi/scripts/plotly_streamlit.py", line 14, in <module>
    from suntime import Sun`

anyone happen to know how to further debug or resolve this?

tia.

**Comment by ehpersonal38** [comment, marked as answer]
Ignore my previous post. This issue is due to the fork you're using not having suntime in the x86 requirements file. https://github.com/MatthewBCooke/BirdNET-Pi/blob/debian/requirements-x86.txt

Please talk to the owner of that fork regarding this issue.

**Reply by kobius77** [reply]
thanks for the hints - strangely it's working now, not sure if it's possibly due to an (automatic?) update? maybe this pulled in the missing module? already asked @MatthewBCooke  in another thread if/how this fork does update.

**Comment by kobius77** [comment]
FYI: after reinstalling (again on the very same Debian11 VM) i ran into the same issue again. finding my own bug-report in here, i just ran the updater within BirdNetPi, which results in the bug message being gone - but the stats not coming up either.

now it's just an empty page below Species Stats.
![image](https://github.com/user-attachments/assets/f1ed197d-629d-481a-926e-5535a953418e)
